INFO - Log de condiciones convertidas a formato pandas
INFO - Inicio del proceso de validación de datos.
INFO - Se encontraron 1064 condiciones.
ERROR - PPA03 >= 6 y P05C01 está en (1,2) o P05C02 está en (1,2,3,4,5,6,98) o P05C03 está en (1,2) o P05C04 está en (1,2,3,4,98) o P05C05 está en (1,2,3,4,5,6,7,8,9,98) o P05C06 está en (1,2,3,4,5,6,7,8,9,10,11,12,98) o P05C07A no es vacío o P05C07B no es vacío o P05C08A está en (1,2,3,4,5,6,7,8,9,10,11,12,98) o P05C08B está en (1,2,3,4,5,6,7,8,9,10,11,12,98) o P05C09A no es vacío o P05C09B no es vacío o P05C09C no es vacío o P05C09D no es vacío o P05C09E no es vacío o P05C09F no es vacío o P05C09G no es vacío o P05C010 está en (1,2): 'NoneType' object is not subscriptable. Error de VO
ERROR - P09F02A = 1 y ((P09F02B es (vacio) y P09F02C es vacio) o (P09F02B = 0) y P09F02C = 0)): 'NoneType' object is not subscriptable. Error de DT
ERROR - P09F02A = 2 y (P09F02B > 0 | P09F02C > 0): 'NoneType' object is not subscriptable. Error de DT
ERROR - P10A03=1 y P10A04 está en (1,2): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - P10A05=1 y P10A06 está en (1,2): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - P10A06=1 Y P10A07 no esta en (1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 98): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - P10A06=2 y P10A07 está en (1,2,3,4,5,6,7,8,9,10,11,12,13,14,98): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - P10A08=1 y P10A09 está en (1,2,3,4,5,6): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - P10A08=2 y P10A09 no está en (1,2,3,4,5,6): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - PPA03 >= 7 Y (P10B01=1 y P10B03 no está en (1,2,3,4,5,6,7,8,9,10,11,12,13)): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de MP
ERROR - PPA03 >= 7 Y (P10B01=2 & P10B03 está en (1,2,3,4,5,6,7,8,9,10,11,12,13)): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de MP
ERROR - PPA03 >= 7 Y (P10B01=1 & (P10B04<0 o P10B04>97)): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de MP
ERROR - PPA03 >= 7 Y (P10B01=2 & (P10B04>0)): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de MP
ERROR - P10C01 no está en (1,2,3) y (P10C12A >=0 o P10C12B > =0): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - P10C01está en (1,2,3) y P10C13 no está en (1,2,3,4,5,6): "P10C01in" is not a supported function. Error de PH
ERROR - PC10C01>0 y P10C15 no esta en (1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13,14, 98): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C16A ESTÁ EN (1,2,) Y P10C16B no esta en (1, 2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C16B ESTÁ EN (1,2) Y P10C16C no esta en (1, 2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C16C ESTÁ EN (1,2) Y P10C16D no esta en (1, 2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C16D ESTÁ EN (1,2) Y P10C16E no esta en (1, 2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C16E ESTÁ EN (1,2) Y P10C17A no esta en (1, 2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C17A ESTÁ EN (1,2) Y P10C17B no esta en (1, 2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C17B ESTÁ EN (1,2) Y P10C17C no esta en (1, 2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C17C ESTÁ EN (1,2) Y P10C17D no esta en (1, 2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C17D ESTÁ EN (1,2) Y P10C17E no esta en (1, 2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C17E ESTÁ EN (1,2) Y P10C17F no esta en (1, 2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C17G ESTÁ EN (1,2) Y P10C18 no esta en (1, 2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21=9 y P10C22 no esta en (1, 2): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - PC10C21<5 y P10C22 no es vacio: 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C23=1 y P10C24A no esta en (1, 2): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - P10C25A = 1 y P10C25B no está (1,2): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de VO
ERROR - P10C21 ESTÁ EN (5,6,7,8) y P10C27 >= 1: 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 ESTÁ EN (5,6,7,8) y P10C29A ESTÁ EN (1,2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 ESTÁ EN (5,6,7,8) y P10C30A ESTÁ EN (1,2): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 ESTÁ EN (5,6,7,8) y (P10C31A ESTÁ EN (1,2) o P10C31B >= 1): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 ESTÁ EN (5,6,7,8) y (P10C33A está en (1,2) o P10C33B >= 1): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 ESTÁ EN (5,6,7,8) y (P10C34A está en (1,2) o P10C34B >= 1): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 ESTÁ EN (5,6,7,8) y (P10C35A está en (1,2) o P10C35B >= 1): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 ESTÁ EN (5,6,7,8) y (P10C36A está en (1,2) o P10C36B >= 1): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 ESTÁ EN (5,6,7,8) y (P10C37A está en (1,2) o P10C37B >= 1): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 ESTÁ EN (5,6,7,8) y (P10C38A está en (1,2) o P10C38B >= 1): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 ESTÁ EN (5,6,7,8) y (P10C39A está en (1,2) o P10C39B >= 1): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 ESTÁ EN (5,6,7,8) y (P10C40A está en (1,2) o P10C40B >= 1): 'NoneType' object is not subscriptable. Error de PH
ERROR - P10C21 < 4 P10C41 no esta en (1, 2): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR -  (P10C21 esta en (5,6,7,8)) y P10C51A no está en (1): only list-like objects are allowed to be passed to isin(), you passed a [int]. Error de PH
ERROR - P10C01 > 1 y PPA03 >= 7 y P10D17 <> 1 y P10D18A no es (vacio): 'NoneType' object is not subscriptable. Error de DB
ERROR - P10E02 = 1 y (P10E03A es vacío o P10E03B es vacío o P10E03C es vacío ).: Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - P10D01=1 y P10E01B es (vacio): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de BG
ERROR - P10E02=2 y (P10E03A>0 o P10E03B>0 o P10E03C>0): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - P10E10A = 1 o P10E10B >= 0: '>=' not supported between instances of 'str' and 'int'. Error de PH
ERROR - P10D01=2 y P10E02 >= 1 y P10E10A no esta en (1, 2, 3, 4, 5, 6, 7): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de PH
ERROR - PPA03 >= 18 (P10C21 está en (5,6) o P10D07 está en (5,6)) y (P14A01 = 2 o P14A01 es vacío): Only named functions are supported. Error de nan
ERROR - PPA03 >= 18 & (P10C21 está en (7,8) o P10D07 está en (7,8)) y (P16A01 = 2 o P16A01 es vacío): "['FECHA'] not in index". Error de nan
ERROR - P13A04C = 1 & P13A07A <> 0 & ((P13A07D = 0 | P13A07D es (vacío): ('EOF in multi-line statement', (2, 0)). Error de LA
ERROR - P13A08 = 1 & (P13A09 es (vacío) | P13A10A es (vacío) | P13A11A es (vacío) | P13A12A = es (vacío)| P13A12B = es (vacío)| P13A12C = es (vacío)| P13A12D = es (vacío)| P13A12E = es (vacío) ): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de LA
ERROR - (P10C21 esta en (5,6) o P10D07 esta en (5,6)) y P14A01 es (vacio): "['FECHA'] not in index". Error de DB
ERROR - P16B08A = 0 & (P16B08C no es (vacio) | P16B08D > 0)): ('EOF in multi-line statement', (2, 0)). Error de KO
ERROR - P16B09A = 0 & (P16B09C no es (vacio) |  P16B09D > 0)): ('EOF in multi-line statement', (2, 0)). Error de KO
ERROR - P16D01 = 2 &  (P16D02 = no es (vacío) |  P16D03 = no es (vacío) | P16D04 = no es (vacío) | P16D05A = no es (vacío) | P16D05B = no es (vacío) | P16D06A = no es (vacío) | P16D06B = no es (vacío) | P16D06C = no es (vacío) | P16D06D = no es (vacío) | P16D06E = no es (vacío) | P16D07 = no es (vacío) | P16D08 = no es (vacío) ): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de LA
ERROR - P17A05 = 2 & (p17A06 no es (vacío)  | P17A06B no es (vacío) |   P17A07 no es (vacío)  | P17A08 no es (vacío)  | P17A09A no es (vacío) |  P17A09B no es (vacío) |  P17A10A no es (vacío) |  P17A10B no es (vacío) |  P17A11A no es (vacío) |  P17A11B no es (vacío) |  P17A12A no es (vacío) |  P17A12B no es (vacío) |  P17A13A no es (vacío) |  P17A13B no es (vacío) |  P17A14A no es (vacío) |  P17A14B no es (vacío) |  P17A14C no es (vacío) |  P17A15A no es (vacío) |  P17A15B no es (vacío) |  P17A16  no es (vacío) | P17A17A no es (vacío) |  P17A17B no es (vacío) |  P17A18  no es (vacío) | P17A19A no es (vacío) |  P17A19B no es (vacío) |  P17A19C no es (vacío) | ): Python keyword not valid identifier in numexpr query (<unknown>, line 1). Error de LA
ERROR - P13A04C = 1 & P13A07A <> 0 & ((P13A07D = 0 | P13A07D es (vacío)): ('EOF in multi-line statement', (2, 0)). Error de LA
ERROR - p13a04C = 2 & p13A08 es (vacío): list index out of range. Error de LA
WARNING - R[write to console]: Loading required package: DBI

WARNING - R[write to console]: Total de inconsistencias: 101

WARNING - R[write to console]: >> InconsistenciasGRUPO11.xlsx -> 3

WARNING - R[write to console]: >> InconsistenciasGRUPO2.xlsx -> 8

WARNING - R[write to console]: >> InconsistenciasGRUPO15.xlsx -> 8

WARNING - R[write to console]: >> InconsistenciasGRUPO32.xlsx -> 4

WARNING - R[write to console]: >> InconsistenciasGRUPO35.xlsx -> 7

WARNING - R[write to console]: >> InconsistenciasGRUPO4.xlsx -> 7

WARNING - R[write to console]: >> InconsistenciasGRUPO8.xlsx -> 5

WARNING - R[write to console]: >> InconsistenciasGRUPO18.xlsx -> 8

WARNING - R[write to console]: >> InconsistenciasGRUPO37.xlsx -> 5

WARNING - R[write to console]: >> InconsistenciasGRUPO23.xlsx -> 4

WARNING - R[write to console]: >> InconsistenciasGRUPO30.xlsx -> 4

WARNING - R[write to console]: >> InconsistenciasGRUPO38.xlsx -> 15

WARNING - R[write to console]: >> InconsistenciasGRUPO34.xlsx -> 4

WARNING - R[write to console]: >> InconsistenciasGRUPO22.xlsx -> 4

WARNING - R[write to console]: >> InconsistenciasGRUPO24.xlsx -> 4

WARNING - R[write to console]: >> InconsistenciasGRUPO7.xlsx -> 3

WARNING - R[write to console]: >> InconsistenciasGRUPO29.xlsx -> 1

WARNING - R[write to console]: >> InconsistenciasGRUPO27.xlsx -> 1

WARNING - R[write to console]: >> InconsistenciasGRUPO12.xlsx -> 3

WARNING - R[write to console]: >> InconsistenciasGRUPO5.xlsx -> 2

WARNING - R[write to console]: >> InconsistenciasGRUPO31.xlsx -> 1

DEBUG - Generated new state WjI2AplnmofXdHeiLH0gin1tpUI4Bg.
INFO - "GET /?state=WjI2AplnmofXdHeiLH0gin1tpUI4Bg&code=4/0AfJohXk7CmwDPcTb_Ko4HzWmhs9O5-sTwXNYYr26f5pxPABgjx7a-zLarsQNg2bir-FQDw&scope=https://www.googleapis.com/auth/drive HTTP/1.1" 200 65
DEBUG - Encoding `client_id` "739655388044-9qdarqeuerp3ffimkljjeh7lfa59ajbk.apps.googleusercontent.com" with `client_secret` as Basic auth credentials.
DEBUG - Requesting url https://oauth2.googleapis.com/token using method POST.
DEBUG - Supplying headers {'Accept': 'application/json', 'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8'} and data {'grant_type': 'authorization_code', 'code': '4/0AfJohXk7CmwDPcTb_Ko4HzWmhs9O5-sTwXNYYr26f5pxPABgjx7a-zLarsQNg2bir-FQDw', 'redirect_uri': 'http://localhost:62573/'}
DEBUG - Passing through key word arguments {'timeout': None, 'auth': <requests.auth.HTTPBasicAuth object at 0x0000023659ACDBD0>, 'verify': True, 'proxies': None, 'cert': None}.
DEBUG - Starting new HTTPS connection (1): oauth2.googleapis.com:443
DEBUG - https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 200 None
DEBUG - Request to fetch token completed with status 200.
DEBUG - Request url was https://oauth2.googleapis.com/token
DEBUG - Request headers were {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': 'application/json', 'Connection': 'keep-alive', 'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8', 'Content-Length': '157', 'Authorization': 'Basic NzM5NjU1Mzg4MDQ0LTlxZGFycWV1ZXJwM2ZmaW1rbGpqZWg3bGZhNTlhamJrLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tOkdPQ1NQWC1yemdKVWdpUDZnVmdtSnl3RUVwbExraUxtNG9v'}
DEBUG - Request body was grant_type=authorization_code&code=4%2F0AfJohXk7CmwDPcTb_Ko4HzWmhs9O5-sTwXNYYr26f5pxPABgjx7a-zLarsQNg2bir-FQDw&redirect_uri=http%3A%2F%2Flocalhost%3A62573%2F
DEBUG - Response headers were {'Date': 'Fri, 29 Sep 2023 16:11:01 GMT', 'Cache-Control': 'no-cache, no-store, max-age=0, must-revalidate', 'Expires': 'Mon, 01 Jan 1990 00:00:00 GMT', 'Pragma': 'no-cache', 'Content-Type': 'application/json; charset=utf-8', 'Vary': 'Origin, X-Origin, Referer', 'Content-Encoding': 'gzip', 'Server': 'scaffolding on HTTPServer2', 'X-XSS-Protection': '0', 'X-Frame-Options': 'SAMEORIGIN', 'X-Content-Type-Options': 'nosniff', 'Alt-Svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000', 'Transfer-Encoding': 'chunked'} and content {
  "access_token": "ya29.a0AfB_byDC0visI0JVseHprUr-v6SGWrX88FTRpvxi6FLIRlklRKS3McNA3kprOIqrFdcAXr8YOudr_DmEwTRLVcr1Sr_NI8OVmWSVcqPiA__ek234WdKMr0CxmAdHaAUYEsKYhvuGwMR__BLmvcFIyF25KRt5-8MrBjcSaCgYKAZMSARISFQGOcNnCHdDG1-_WvPfDnZtRDDlCnQ0171",
  "expires_in": 3599,
  "refresh_token": "1//053tB1fd6_a1oCgYIARAAGAUSNwF-L9IrpdOBot1G6YlgkgjiLJkQpypQ8Tc0uOul_2RvTjTpIzOaHWV3XlEHgAXTRLlt22tmgCM",
  "scope": "https://www.googleapis.com/auth/drive",
  "token_type": "Bearer"
}.
DEBUG - Invoking 0 token response hooks.
DEBUG - Obtained token {'access_token': 'ya29.a0AfB_byDC0visI0JVseHprUr-v6SGWrX88FTRpvxi6FLIRlklRKS3McNA3kprOIqrFdcAXr8YOudr_DmEwTRLVcr1Sr_NI8OVmWSVcqPiA__ek234WdKMr0CxmAdHaAUYEsKYhvuGwMR__BLmvcFIyF25KRt5-8MrBjcSaCgYKAZMSARISFQGOcNnCHdDG1-_WvPfDnZtRDDlCnQ0171', 'expires_in': 3599, 'refresh_token': '1//053tB1fd6_a1oCgYIARAAGAUSNwF-L9IrpdOBot1G6YlgkgjiLJkQpypQ8Tc0uOul_2RvTjTpIzOaHWV3XlEHgAXTRLlt22tmgCM', 'scope': ['https://www.googleapis.com/auth/drive'], 'token_type': 'Bearer', 'expires_at': 1696007629.5902882}.
INFO - file_cache is only supported with oauth2client<4.0.0
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
DEBUG - URL being requested: POST https://www.googleapis.com/upload/drive/v3/files?alt=json&uploadType=multipart
